### PR TITLE
Create Endpoint for exporting a bucket or a set of objects as a ProActive package

### PR DESCRIPTION
**Pull request by amouhoub**

### CODE REVIEW

File: src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java [hunk 1]
```diff
@@ -519,6 +519,74 @@ public ResponseEntity<List<CatalogObjectMetadata>> list(
         }
     }
 
+    @Operation(summary = "Export catalog objects as a zipped package", description = "Note: Returns catalog objects metadata associated to the latest revision.")
```
Comment (aminelouati) [inline src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java:522]: I propose to briefly define what we mean by "package"

**Comment by fviale** [comment]
Retest this please

**Review by aminelouati — APPROVED** [review]

**Comment by amouhoub** [comment]
retest this please

**Comment by amouhoub** [comment]
retest this please

**Review by nebilbenmabrouk — APPROVED** [review]
LGTM. Minor comments about sonar issues

**Pull request merged**